src/main/java/org/cascadebot/cascadebot/commands/music/VolumeCommand.java

```diff
@@ -16,23 +16,21 @@ public void onCommand(Member sender, CommandContext context) {
             context.getTypedMessaging().replyInfo("Current volume is %s%%", player.getPlayer().getVolume());
             return;
         }
-        Integer Volume = null;
-        try {
-            Volume = context.getArgAsInteger(0);
-
-        } catch (java.lang.NumberFormatException e) {
+        int volume = -1;
+        if(context.isArgInteger(0)) {
+            volume = context.getArgAsInteger(0);
             context.getUIMessaging().replyUsage(this);
             return;
         }
-        if (Volume < 0 || Volume > 100) {
+        if (volume < 0 || volume > 100) {
             context.getTypedMessaging().replyWarning("Volume needs to be between 100 and 0");
```

Comment:
Swap these two round i.e `between 0 and 100`

Reply:
Umm, is it not that already?

Reply:
in the text not the if statment

Reply:
In fact switch the if statement back around because now thats broken

Reply:
The if statement seems fine to me?

Reply:
Whops nvm